src/components/Books.js

```diff
@@ -1,17 +1,35 @@
+/* eslint-disable no-nested-ternary */
 /* eslint-disable react/prop-types */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable **all** linter checks on your code to ensure that your code is robust code and clean.